### DISCUSSION TITLE
Popovers

### DISCUSSION THREAD

**Discussion by mauricekindermann** [livewire/flux#198] [Feature Requests] [answered]
Popovers are somewhere between a Modal, Tooltip, and a Dropdown. 

They are positioned like a Tooltip (top, bottom, sides, etc), open like a Dropdown (relative to clicked/parent) and behave like a Modal (dismiss by clicking outside, interact with inside). 

They would be a very nice additional to Flux, and seemingly all the pieces are already there. 

Wikipedia use them for inline-hover context.

![image](https://github.com/user-attachments/assets/66eff32a-a5c2-43be-838a-33d9aee6a010)

I've used them via Bootstrap for years, here's one quick example

![popovers](https://github.com/user-attachments/assets/e371c983-0450-4bf9-8d84-12a5927e0e1f)

Twitter use them for many things

![image](https://github.com/user-attachments/assets/948addb6-aa79-41b7-91c7-68b3ed1896bd)

Facebook as well

![image](https://github.com/user-attachments/assets/044015bb-2805-4405-a8cd-6d4cc54c88a6)

You get the idea!

**Comment by joshhanley** [comment, marked as answer]
These are now available as "info tooltips". See here in the docs https://fluxui.dev/components/tooltip#info-tooltip
